Styles/main.css

```diff
@@ -1,85 +1,112 @@
+h1 {
+    background-color: #8E443D;
+    color: transparent;
+    background-image: radial-gradient(circle, rgba(2, 0, 36, 1) 0%, rgba(73, 9, 121, 1) 18%, rgba(185, 36, 111, 1) 100%);
+    background-repeat: repeat-x;
+    font-family: "Rubik", sans-serif;
+    text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.2);
+    -webkit-background-clip: text;
+    background-clip: text;
+}

```

Comment:
I've made a number of formatting edits here for you.  Try to keep your indendation consistent, and put each CSS declaration on its own line